certbot/certbot/_internal/main.py

```diff
@@ -921,6 +922,62 @@ def update_account(config: configuration.NamespaceConfig,
     return None
 
 
+def fetch_account(config: configuration.NamespaceConfig,
+                   unused_plugins: plugins_disco.PluginsRegistry) -> Optional[str]:
+    """Fetch account info on the server.
+
+    :param config: Configuration object
+    :type config: configuration.NamespaceConfig
+
+    :param unused_plugins: List of plugins (deprecated)
+    :type unused_plugins: plugins_disco.PluginsRegistry
+
+    :returns: `None`
+    :rtype: None
+
+    """
+    # Portion of _determine_account logic to see whether accounts already
+    # exist or not.
+    account_storage = account.AccountFileStorage(config)
+    accounts = account_storage.find_all()
+
+    if not accounts:
+        return "Could not find an existing account to fetch."
+
+    acc, acme = _determine_account(config)
+    cb_client = client.Client(config, acc, None, None, acme=acme)
+
+    if not cb_client.acme:
+        raise errors.Error("ACME client is not set.")
+
+    regr = cb_client.acme.query_registration(acc.regr)
+
+    if config.verbose_count > 0:
+        display_util.notify(f"Account URI: {regr.uri}")
+
+        thumbprint = jose.b64encode(regr.body.key.thumbprint(
+            hash_function=SHA256)).decode()
+        display_util.notify(f"Account thumbprint: {thumbprint}")
+
+    phones = []
+    emails = []
+
+    for contact in regr.body.contact:
+        if contact.startswith('tel:'):
+            phones.append(contact[4:])
+        if contact.startswith('mailto:'):
+            emails.append(contact[7:])
+
+    display_util.notify("Phone number{} associated with account: {}".format(
+                            "s" if len(phones) > 1 else "",
+                            ", ".join(phones) if len(phones) > 0 else "none"))
```

Comment:
I'm not sure that it makes sense to handle `tel:` contacts, since Certbot doesn't support them in the `register` and `update_account` verbs. I do see there's some code for it in `acme`.

For most (all?) users, the output will superfluously tell them they don't have any phone contacts.

What do you think about cutting this?

Reply:
Personally, I don't think certbot should be a Let's Encrypt only client. For example, Buypass (https://api.buypass.com/acme/directory) supports setting the phone number as an account contact. (Just setting `phone = "+1-415-555-0101"` in `acme/acme/messages.py` to hardcode a phone number was perfectly accepted by the Buypass ACME API.)

That said, I haven't seen any ACME *client* supporting phone numbers (searched for `phone` in some of the most popular ACME client Github repositories). So my toughts here can go two ways:

* There's at least *one* ACME *server* supporting phone contacts, so that may warrant including it in this PR;
* There's no ACME *client* supporting phone contacts, so even if ACME APIs are accepting it, it couldn't have been used anyway, so adding it here would never result in anything. However, I have just checked *a few* clients, maybe there are ACME clients out there supporting phone contacts, I dunno..

Personally I don't think it's much of a problem leaving it in here, even if it's superfluous for probably 100&nbsp;% of time, for just in case :stuck_out_tongue:

Reply:
>Personally, I don't think certbot should be a Let's Encrypt only client

Agreed. 

If Certbot was to support phone contacts, I think we should do this by enhancing `register`/`update_account`/`show_account` as a single piece of functionality.

Unless there is a use case where a phone contact is set out-of-band (e.g. web GUI) I am more strongly leaning towards not showing that information here. It might raise more confusing questions ("well how do I set a phone number?").

Reply:
I could just comment out the phone stuff, waiting for the enhancement :)

Reply:
I think we'd like to avoid having commented code in the codebase, because it gets stale/wrong pretty quickly. It will be preserved in this PR anyway.